GetandTake/Program.cs

```diff
@@ -1,8 +1,14 @@
 using AutoMapper;
 using GetandTake.Configuration;
 using GetandTake.Services.AutoMapper;
+using static System.Net.Mime.MediaTypeNames;
 
 var builder = WebApplication.CreateBuilder(args);
+builder.Host.ConfigureLogging(logging =>
```

Comment:
Let's move logger configuration to separate extension method of `builder.Host` like you did for `RegisterServices`